dffml/operation/mapping.py

```diff
@@ -16,6 +16,50 @@
     outputs={"value": MAPPING_VALUE},
 )
 def mapping_extract_value(mapping: Dict[str, Any], traverse: List[str]):
+    """
+    Extracts value from a given mapping.
+
+    Parameters
+    ++++++++++
+    mapping : dict
+        The mapping to extract the value from.
+    traverse : list[str]
+        A list of keys to traverse through the mapping dictionary and extract the values.
+
+    Returns
+    +++++++
+    dict
+        A dictionary containing the value of the keys.
+
+    Examples
+    ++++++++
+
+    >>> dataflow = DataFlow.auto(mapping_extract_value, GetSingle)
+    >>>
+    >>> dataflow.seed.append(
+    ...     Input(
+    ...         value=[mapping_extract_value.op.outputs["value"].name],
+    ...         definition=GetSingle.op.inputs["spec"],
+    ...     )
+    ... )
+    >>> Inputs = [
```

Comment:
Minor thing, we never camelcase variables

```suggestion
    >>> inputs = [
```